scripts/diffThemes.ts

```diff
@@ -0,0 +1,66 @@
+import {PrimerStyleDictionary} from '~/src/PrimerStyleDictionary'
+// @ts-expect-error: no types exists for style-dictionary createDictionary
+import createDictionary from 'style-dictionary/lib/utils/createDictionary'
+import {themes as themesConfigArray} from './themes.config'
+import type StyleDictionary from 'style-dictionary'
+
+PrimerStyleDictionary.registerTransform({
+  type: `value`,
+  transitive: true,
+  name: `valueToNull`,
+  transformer: _token => null,
+})
+
+const tokenNameArray = ({dictionary}: {dictionary: StyleDictionary.Dictionary}) =>
+  dictionary.allTokens.map(({name}: {name: string}) => name)
+
+const themes = themesConfigArray.map(({filename, source, include}) => {
+  const sd = PrimerStyleDictionary.extend({
+    source,
+    include,
+    platforms: {
+      json: {
+        transforms: ['name/pathToDotNotation'],
+      },
+    },
+  })
+  return [
+    filename,
+    tokenNameArray({
+      dictionary: createDictionary({properties: sd.exportPlatform('json')}),
+    }),
+  ]
+})
+
+/**
+ *
+ * @param mainThemeName - name of the theme (from themes.config.ts) to compare all other themes against
+ */
+
+const diffThemes = (mainThemeName: string) => {
```

Comment:
Would be cool to have some tests for this with happy/unhappy path data to confirm it works as expected.. but from scanning the code it looks good 👍

Reply:
Yes, I will look into this as a separate PR. 👍